spec/requests/api/v2/session_spec.rb

```diff
@@ -95,6 +105,14 @@
           expect(user.last_name).to eq "updated last_name"
         end
       end
+
+      context 'on first login' do
+        let(:user) { create(:user, first_login_date_utc: nil) }
+
+        it 'Sends a welcome email to the user' do
+          expect(govuk_notify_request).to have_been_made
+        end
+      end
```

Comment:
Do we need this too?
```
`on second login`
it 'doesn’t send an email'
```

Reply:
flavour this wording...
`on subsequent login`